crates/rspack_core/src/utils/visitor.rs

```diff
@@ -225,6 +225,10 @@ pub fn extract_member_expression_chain<'e, T: Into<MaybeExpr<'e>>>(
       Expr::Member(ref expr) => {
         walk_member_expr(expr, false, members, members_optionals, members_spans, kind)
       }
+      Expr::This(ref this_expr) => {
```

Comment:
Add `this` expression handler to add in front of the member chain, otherwise it can not be sliced correctly